timemachine/md/align.py

```diff
@@ -0,0 +1,98 @@
+import functools
+from typing import Tuple
+
+import jax
+import jax.numpy as jnp
+import numpy as np
+from numpy.typing import NDArray
+from rdkit import Chem
+
+from timemachine.fe import topology
+from timemachine.fe.utils import get_mol_masses, get_romol_conf
+from timemachine.ff import Forcefield
+from timemachine.integrator import LangevinIntegrator
+from timemachine.potentials import generic
+
+
+def align_mols_by_core(
+    mol_a: Chem.Mol, mol_b: Chem.Mol, core: NDArray, ff: Forcefield, n_steps: int = 200, k: float = 10000
+) -> Tuple[NDArray, NDArray]:
+    """
+    Given two mols and a core mapping, simulate the mols in vacuum with harmonic bounds between them to get aligned poses.
```

Comment:
bounds -> bonds

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/813/commits/c45d947dbfd2941220a5e5959ed9dd3c1b623cc3